src/utils/resources/bootableresources/hooks/useAccessCreateBootableVolume.ts

```diff
@@ -0,0 +1,35 @@
+import { PersistentVolumeClaimModel } from '@kubevirt-ui/kubevirt-api/console';
+import DataImportCronModel from '@kubevirt-ui/kubevirt-api/console/models/DataImportCronModel';
+import DataSourceModel from '@kubevirt-ui/kubevirt-api/console/models/DataSourceModel';
+import { K8sVerb, useAccessReview } from '@openshift-console/dynamic-plugin-sdk';
+
+type UseCanCreateBootableVolume = (
+  namespace: string,
+) => [canCreateDS: boolean, canCreatePVC: boolean, loading: boolean];
+
+const useCanCreateBootableVolume: UseCanCreateBootableVolume = (namespace) => {
+  const [canCreatePVC, loadingPVC] = useAccessReview({
+    group: PersistentVolumeClaimModel.apiGroup,
+    namespace: namespace,
+    resource: PersistentVolumeClaimModel.plural,
+    verb: 'create' as K8sVerb,
+  });
+
+  const [canCreateDS, loadingDS] = useAccessReview({
+    group: DataSourceModel.apiGroup,
+    namespace: namespace,
+    resource: DataSourceModel.plural,
+    verb: 'create' as K8sVerb,
+  });
+
+  const [canCreateDIC, loadingDIC] = useAccessReview({
+    group: DataImportCronModel.apiGroup,
+    namespace: namespace,
+    resource: DataImportCronModel.plural,
+    verb: 'create' as K8sVerb,
+  });
+
+  return [canCreateDS && canCreateDIC, canCreatePVC, loadingPVC || loadingDS || loadingDIC];
```

Comment:
loading shoul be &&
true | true | false will give true and u want to give false..
true & true & false will give right false

Reply:
With `useAccessReview` you don't get the `loaded` boolean as in `useK8sWatchResource`.
With this hook, you get `loading` as a boolean. So yes, it's an OR operator.

I want that the result is false when all the loading is false. It means that all the access has been fetched successfully